projects/packages/videopress/src/client/block-editor/blocks/video/components/banner/style.scss

```diff
@@ -0,0 +1,29 @@
+.block-banner {
+	display: flex;
```

Comment:
The code is misaligned here. 💅🏽

Reply:
Solved moving indentation from spaces to tabs.

Reply:
Thanks @lhkowalski